homeassistant/components/airzone/coordinator.py

```diff
@@ -23,7 +23,7 @@ class AirzoneUpdateCoordinator(DataUpdateCoordinator):
 
     def __init__(self, hass: HomeAssistant, airzone: AirzoneLocalApi) -> None:
         """Initialize."""
-        self.airzone = airzone
+        self.airzone: AirzoneLocalApi = airzone
```

Comment:
Isn't this already inferred?

Reply:
Reverted change in 66bd22912bbcf6577569050a66093a5dd4394fab